resources/resincli.robot

```diff
@@ -69,6 +69,15 @@ Device ${device_uuid} log should contain ${value}
     Process ${result}
     Should Contain    ${result.stdout}    ${value}
 
+Application ${application_name} should contain variable ${variable_name} with value ${variable_value}
```

Comment:
This seems wrong - we should have one keyword to add an ENV variable and a separate one for checking if an ENV variable is present.